sympy/matrices/tests/test_eigen.py

```diff
@@ -493,13 +498,19 @@ def test_definite():
     m = Matrix([[2, -1, 0], [-1, 2, -1], [0, -1, 2]])
     assert m.is_positive_definite == True
     assert m.is_positive_semidefinite == True
+    assert _is_positive_semidefinite_by_cholesky_factorization_with_pivots(m)
+    assert _is_positive_semidefinite_by_eigenvalues(m)
+    assert _is_positive_semidefinite_by_minors(m) 
```

Comment:
There is a trailing whitespace at the end of this line which is causing quality tests to fail, please fix that.

Reply:
My bad there. I removed whitespace and updated the flake8 part of setup.cfg to catch it earlier next time.

Reply:
I guess adding whitespace to setup.cfg was a bad idea, since it caused a whole bunch of linting errors in the pipeline. Removed in latest commit.